indexer/packages/postgres/src/models/order-model.ts

```diff
@@ -92,7 +92,7 @@ export default class OrderModel extends BaseModel {
         goodTilBlock: { type: ['string', 'null'], default: null, pattern: IntegerPattern },
         goodTilBlockTime: { type: ['string', 'null'], default: null, format: 'date-time' },
         createdAtHeight: { type: ['string', 'null'], default: null, pattern: IntegerPattern },
-        clientMetadata: { type: 'string', pattern: IntegerPattern },
+        clientMetadata: { type: ['string', 'null'], pattern: IntegerPattern },
```

Comment:
I think you'll need to add a file like https://github.com/dydxprotocol/v4-chain/blob/main/indexer/packages/postgres/src/db/migrations/migration_files/20230705221636_add_fills_client_metadata.ts

Reply:
I reverted this as this change wasn't needed since we don't upsert any liquidation order. Only the fill model needed updating.

Also the fill table already is `nullable`, from the testnet DB:
```
dydx=> \d fills
                                    Table "public.fills"
     Column      |           Type           | Collation | Nullable |        Default
-----------------+--------------------------+-----------+----------+------------------------
 id              | uuid                     |           | not null |
 subaccountId    | uuid                     |           | not null |
 side            | text                     |           | not null |
 liquidity       | text                     |           | not null |
 type            | text                     |           | not null |
 clobPairId      | bigint                   |           | not null |
 orderId         | uuid                     |           |          |
 size            | numeric                  |           | not null |
 price           | numeric                  |           | not null |
 quoteAmount     | numeric                  |           | not null |
 eventId         | bytea                    |           | not null |
 transactionHash | character varying(255)   |           | not null |
 createdAt       | timestamp with time zone |           | not null |
 createdAtHeight | bigint                   |           | not null |
 clientMetadata  | bigint                   |           |          |
 fee             | character varying(255)   |           | not null | '0'::character varying
Indexes:
    "fills_pkey" PRIMARY KEY, btree (id)
    "fills_clobPairId_createdAtHeight_partial" btree ("clobPairId", "createdAtHeight") WHERE liquidity = 'TAKER'::text
    "fills_clobpairid_createdat_index" btree ("clobPairId", "createdAt")
    "fills_orderid_index" btree ("orderId")
    "fills_subaccountid_clobpairid_index" btree ("subaccountId", "clobPairId")
    "fills_subaccountid_createdat_index" btree ("subaccountId", "createdAt")
    "fills_subaccountid_createdatheight_index" btree ("subaccountId", "createdAtHeight")
Check constraints:
    "fills_liquidity_check" CHECK (liquidity = ANY (ARRAY['TAKER'::text, 'MAKER'::text]))
    "fills_side_check" CHECK (side = ANY (ARRAY['BUY'::text, 'SELL'::text]))
    "fills_type_check" CHECK (type = ANY (ARRAY['MARKET'::text, 'LIMIT'::text, 'LIQUIDATED'::text, 'LIQUIDATION'::text]))
Foreign-key constraints:
    "fills_createdatheight_foreign" FOREIGN KEY ("createdAtHeight") REFERENCES blocks("blockHeight")
    "fills_eventid_foreign" FOREIGN KEY ("eventId") REFERENCES tendermint_events(id)
    "fills_orderid_foreign" FOREIGN KEY ("orderId") REFERENCES orders(id)
    "fills_subaccountid_foreign" FOREIGN KEY ("subaccountId") REFERENCES subaccounts(id)
```